src/runtime/components/AntTag.vue

```diff
@@ -24,34 +23,53 @@ const classes = computed(() => {
   const variants: Record<TagColorType, string> = {
     [TagColorType.danger]: 'bg-danger-500 text-danger-500-font',
     [TagColorType.info]: 'bg-info-500 text-info-500-font',
+    [TagColorType.primary]: 'bg-primary-500 text-primary-500-font',
+    [TagColorType.secondary]: 'bg-secondary-500 text-secondary-500-font',
     [TagColorType.base]: 'bg-neutral-300 text-neutral-300-font',
     [TagColorType.success]: 'bg-success-500 text-success-500-font',
     [TagColorType.warning]: 'bg-warning-500 text-warning-500-font',
   };
   return {
     'rounded-md inline-flex items-center': true,
-		'px-1 py-0.5 text-xs gap-0.5': props.size === AntTagSize.xs3,
-		'px-1.5 py-1 text-xs gap-1': props.size === AntTagSize.xs2,
-		'px-2 py-1.5 text-xs gap-1.5': props.size === AntTagSize.xs,
-		'px-2 py-1.5 text-sm gap-1.5': props.size === AntTagSize.sm,
-		'px-2.5 py-2 text-sm gap-2': props.size === AntTagSize.md,
-		'px-3 py-2.5 text-sm gap-2.5': props.size === AntTagSize.lg,
-		[variants[props.colorType]]: true,
+    'px-1 py-0.5 text-xs gap-0.5': props.size === AntTagSize.xs3,
+    'px-1.5 py-1 text-xs gap-1': props.size === AntTagSize.xs2,
+    'px-2 py-1.5 text-xs gap-1.5': props.size === AntTagSize.xs,
+    'px-2 py-1.5 text-sm gap-1.5': props.size === AntTagSize.sm,
+    'px-2.5 py-2 text-sm gap-2': props.size === AntTagSize.md,
+    'px-3 py-2.5 text-sm gap-2.5': props.size === AntTagSize.lg,
+    [variants[props.colorType]]: true,
   };
 });
 const getIconSize = computed(() => {
-	if(props.size === AntTagSize.xs || props.size === AntTagSize.xs2 || props.size === AntTagSize.xs3) {
-		return IconSize.xs
-	} else {
-		return IconSize.sm
-	}
-})
+  if (props.size === AntTagSize.xs || props.size === AntTagSize.xs2 || props.size === AntTagSize.xs3) {
+    return IconSize.xs;
+  } else {
+    return IconSize.sm;
+  }
+});
+const iconColorType = computed(() => {
+  if (props.colorType === TagColorType.info) {
```

Comment:
You could use a switch case here instead of this if else statements.
It would be easier to read in my opinion.